### PR TITLE
added solution to HW Lottery

### PR DESCRIPTION
**Pull request by OlehMytiai**

### CODE REVIEW

File: src/main/java/core/basesyntax/Application.java [hunk 1]
```diff
@@ -3,5 +3,14 @@
 public class Application {
     public static void main(String[] args) {
         // create three balls using class Lottery and print information about them in console
```
Comment (AlexRozv) [inline src/main/java/core/basesyntax/Application.java:5]: let's remove this comment

File: src/main/java/core/basesyntax/Application.java [hunk 2]
```diff
@@ -3,5 +3,14 @@
...
+        Lottery lottery = new Lottery();
+        Ball firstBall = lottery.getRandomBall();
+        Ball secondBall = lottery.getRandomBall();
+        Ball thirdBall = lottery.getRandomBall();
+
```
Comment (AlexRozv) [inline src/main/java/core/basesyntax/Application.java:10]: let's remove redundant empty lines

File: src/main/java/core/basesyntax/ColorSupplier.java [hunk 3]
```diff
@@ -1,7 +1,11 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
+    private static final Random RANDOM = new Random();
```
Comment (AlexRozv) [inline src/main/java/core/basesyntax/ColorSupplier.java:6]: it is unnecessary to make it a const

File: src/main/java/core/basesyntax/Lottery.java [hunk 4]
```diff
@@ -0,0 +1,10 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        ColorSupplier colorSupplier = new ColorSupplier();
+        return new Ball(colorSupplier.getRandomColor(), new Random().nextInt(100));
```
Comment (AlexRozv) [inline src/main/java/core/basesyntax/Lottery.java:8]: why did you create an instance of ColorSupplier, but use Random directly in the return statement? Let's keep our code consistent and use either one of those approaches. (Or create instances for both ColorSupplier and Random, or use both of them directly in return)

**Review by AlexRozv — CHANGES_REQUESTED** [review]

File: src/main/java/core/basesyntax/Lottery.java (hunk 4 above)
Comment (mytiaiov) [reply src/main/java/core/basesyntax/Lottery.java]: Copy that

File: src/main/java/core/basesyntax/Application.java [hunk 5]
```diff
@@ -2,6 +2,12 @@
...
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery lottery = new Lottery();
+        Ball firstBall = lottery.getRandomBall();
+        Ball secondBall = lottery.getRandomBall();
+        Ball thirdBall = lottery.getRandomBall();
+        System.out.println("1st " + firstBall.toString());
+        System.out.println("2nd " + secondBall.toString());
+        System.out.println("3rd " + thirdBall.toString());
```
Comment (AlexRozv) [inline src/main/java/core/basesyntax/Application.java:11]: let's use for loop here to create and print balls

**Review by AlexRozv — CHANGES_REQUESTED** [review]

File: src/main/java/core/basesyntax/Application.java (hunk 5 above)
Comment (mytiaiov) [reply src/main/java/core/basesyntax/Application.java:11]: Is it a good practice to create const variable for count of balls?
Example:
`private static final int NUMBER_OF_BALLS = 3;`

**Review by AlexRozv — APPROVED** [review]
good job 👍

**Pull request closed without merging**